test/parallel/test-timers-clear-timeout-interval-equivalent.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+const common = require('../common');
+
+// This test makes sure that timers created with setTimeout can be disarmed by
+// clearInterval and that timers created with setInterval can be disarmed by
+// clearTimeout.
+//
+// This behavior is documented in Node.js' documentation and the HTML Living
+// standard:
+//
+// * Refs: https://html.spec.whatwg.org/multipage/timers-and-user-prompts.html#dom-setinterval
+// * Refs: https://nodejs.org/api/timers.html#timers_class_timeout
+
+const assert = require('assert');
+const Timer = require('timers');
```

Comment:
We don't import `timers` in our timer tests. You can just use the global versions.